crates/sui-framework/sources/crypto.move

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module sui::crypto {
+    /// TO verify signature, if valid, return recovered public key, otherwise throw error.
```

Comment:
- typo: `To`

- Please add explanation on how ecrecover works, and mention it only applies to ECDSA k1 specifically